config/test/affine.yaml

```diff
@@ -3,7 +3,7 @@ train:
   backbone:
     name: "global"
     num_channel_initial: 1
-    extract_levels: [0, 1, 2, 3, 4]
+    depth: 4
```

Comment:
this is not clear and over-constrained - user may want only high-res layers OR only low-res layers

Reply:
this is the same behaviour as before https://github.com/DeepRegNet/DeepReg/blob/main/deepreg/model/backbone/global_net.py#L64

you encode the image until a certain level, then you take the bottom to produce affine.

If we want more functionalities, it can be addressed in another issue/PR.

Reply:
for local-net, the original behaviours has freedom to use any combinations of these levels, e.g. [0], [0,1,2], [0,2,4], [1,2,3,4], [4]........
for global-net, how is depth or extract_levels relevant?

Reply:
local net still can do this, global net has no need of extraction levels,

The point here is that this PR is not changing the existing behaviours, if you want extract_levels to impact the behavior, it should be in a new issue.

In main, only the max of extract_levels is useful.